webapp/src/components/data_selectors/backend_selector.tsx

```diff
@@ -160,7 +160,7 @@ export default class BackendSelector extends React.PureComponent<Props, State> {
 
             // option's label hasn't been fetched yet
             return {
-                label: v,
+                label: v.displayName,
                 value: v,
             };
```

Comment:
@sanjaydemansol Can you please explain this fix? All other types of selectors (non-user) seem to be working correctly. Do they actually need to have this change (which they do receive the change if it is applied in this file)? It seems the problem is specific to users, so I would expect the fix to go into `jira_user_selector.tsx`.

Can you please describe how you discovered the root cause, and how this fixes it? What is the error printed to the console during the crash, and what line does it occur on?

Reply:
@mickmister this is unrelated to the issue. I'm not sure why it is here. I have updated the code to remove this.  The crash is caused by the `assignee` property not being reset. I am awaiting dipak to finish the qa on this one.